index.html

```diff
@@ -1,179 +1,254 @@
     <title>Document</title>
-    <script src="https://kit.fontawesome.com/866fce4fcb.js" crossorigin="anonymous"></script>
-    <link rel="preconnect" href="https://fonts.gstatic.com">
-    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="style.css">
-</head>
-<body>
+    <script
+      src="https://kit.fontawesome.com/866fce4fcb.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="wrap">
+      <a href="#" class="arrow-up"
+        ><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-arrow-up-circle-fill" viewBox="0 0 16 16">
+  <path d="M16 8A8 8 0 1 0 0 8a8 8 0 0 0 16 0zm-7.5 3.5a.5.5 0 0 1-1 0V5.707L5.354 7.854a.5.5 0 1 1-.708-.708l3-3a.5.5 0 0 1 .708 0l3 3a.5.5 0 0 1-.708.708L8.5 5.707V11.5z"/>
+</svg></a>
 
-<div class="wrap">
-    <a href="#" class="arrow-up-a"><i class="fas fa-arrow-up arrow-up"></i></a>
-
-    <div class="menu">
+      <div class="menu">
         <div class="navigation">
-            <p class="navigation-text">Christmas Gift</p>
-            <nav class="menu-nav">
-                <ul>
-                    <li class="navigation-li"><a href="#" class="navigation-a">Home</a></li>
-                    <li class="navigation-li"><a href="#share" class="navigation-a">Share</a></li>
-                    <li class="navigation-li"><a href="#decoration" class="navigation-a">Decorations</a></li>
-                    <li class="navigation-li"><a href="#accessory" class="navigation-a">Accessory</a></li>
-                </ul>
-                <div class="change-theme">
-                    <div class="switch"></div>
-                </div>
-            </nav>
-            <i class="fas fa-th-large bars"></i>
+          <h3 class="navigation-text">Christmas Gift</h3>
+          <nav class="menu-nav">
+            <ul>
+              <li class="navigation-li">
+                <a href="#" class="navigation-a">Home</a>
+              </li>
+              <li class="navigation-li">
+                <a href="#share" class="navigation-a">Share</a>
+              </li>
+              <li class="navigation-li">
+                <a href="#decoration" class="navigation-a">Decorations</a>
+              </li>
+              <li class="navigation-li">
+                <a href="#accessory" class="navigation-a">Accessory</a>
+              </li>
+            </ul>
+            <div class="change-theme">
+              <div class="switch">
+            </div>
+          </nav>
+          <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-list bars" viewBox="0 0 16 16">

```

Comment:
You only have styles for the `bars` class
```suggestion
          <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bars" viewBox="0 0 16 16">

```

Reply:
Right, I took icon and haven't noticed that it's here 😕